src/time.rs

```diff
@@ -51,10 +51,8 @@ impl ClockId {
     #[cfg(not(any(
         target_os = "macos",
         target_os = "ios",
-        all(
-            not(any(target_env = "uclibc", target_env = "newlibc")),
```

Comment:
I'm fine with dropping `uclibc` here since we don't support it at the moment, but it's worth noting #1603 may need to put it back.